src/main/kotlin/uk/gov/gdx/datashare/services/OutboundEventQueueService.kt

```diff
@@ -116,7 +116,7 @@ class OutboundEventQueueService(
         Tag.builder().tagKey("Environment").tagValue(environment).build(),
         Tag.builder().tagKey("Owner").tagValue("gdx-dev-team@digital.cabinet-office.gov.uk").build(),
         Tag.builder().tagKey("Product").tagValue("Government Data Exchange").build(),
-        Tag.builder().tagKey("Repository").tagValue("https://github.com/alphagov/gdx-data-share-poc").build(),
+        Tag.builder().tagKey("Repository").tagValue("https://github.com/alphagov/di-data-life-events-platform").build(),
```

Comment:
I think this will fail a test

Reply:
We'll also possibly want to go and amend any queues we've got already created